autoload/ale/sign.vim

```diff
@@ -361,6 +390,7 @@ function! ale#sign#GetSignCommands(buffer, was_sign_set, sign_map) abort
     if l:is_dummy_sign_set && !g:ale_sign_column_always
         call add(l:command_list, 'sign unplace '
         \   . g:ale_sign_offset
+        \   . s:GroupCmd()
```

Comment:
The use of `priority` in the unplace command here might be what's breaking things. Same as above.

Reply:
Thanks, the problem was that sign-group and sign-priority are supported on vim 8.1.0614 onward but the tests use 8.1.0519 causing all sign commands to fail. I changed the version test to include the patch version and everything seems to be working.

Also had to modify all tests to also consider the sign group.